src/components/ModalEdit/index.js

```diff
@@ -0,0 +1,133 @@
+import { useContext } from "react";
+import DatePicker from "react-datepicker";
+import { Modal, Button } from "react-bootstrap";
+import DoctorsContext from "../../contexts/DoctorsContext";
+
+const ModalEdit = ({
+  setItemToEdit,
+  updateVisit,
+  editItem,
+  errors,
+  setErrors,
+}) => {
+  const setEditedVisitPartials = (visitFieldData) => {
+    setItemToEdit({ ...editItem, ...visitFieldData });
+  };
+
+  const doctors = useContext(DoctorsContext);
+  let { patient_name, text, date, doctor } = editItem;
```

Comment:
why do you have a prop mutation?

Reply:
because i need to set date in correct format

Reply:
mutations in react are bad practices. nobody does that